browser/ui/views/tabs/brave_new_tab_button.cc

```diff
@@ -45,6 +38,22 @@ SkPath BraveNewTabButton::GetBorderPath(const gfx::Point& origin,
   return path;
 }
 
+gfx::Size BraveNewTabButton::CalculatePreferredSize() const {
+  // Overriden so that we use Brave's custom button size
+  gfx::Size size = kButtonSize;
+  const auto insets = GetInsets();
+  size.Enlarge(insets.width(), insets.height());
+  return size;
+}
+
+SkPath BraveNewTabButton::GetBorderPath(const gfx::Point& origin,
+                                        float scale,
+                                        bool extend_to_top) const {
+  return BraveNewTabButton::GetBorderPath(origin, scale, extend_to_top,
```

Comment:
nit: we can just call w/o `BraveNewTabButton`.
Usually we omit class name when calling same class's method.

Reply:
:+1: cheers for the tip!